package.json

```diff
@@ -1,61 +1,28 @@
 {
   "name": "lint-staged",
   "description": "Lint files staged by git",
-  "main": "index.js",
-  "bin": {
-    "lint-staged": "index.js"
-  },
+  "license": "MIT",
+  "repository": "https://github.com/okonet/lint-staged",
+  "author": "Andrey Okonetchnikov <andrey@okonet.ru>",
+  "maintainers": [
+    "Lufty Wiranda <lufty.wiranda@gmail.com",
+    "Suhas Karanth <sudo.suhas@gmail.com"
+  ],
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=4.2.0"
   },
+  "bin": "index.js",
   "files": [
     "src"
   ],
   "scripts": {
-    "commit": "git-cz",
+    "precommit": "remove-lockfiles && node index.js",
+    "cz": "git-cz",
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
-    "pre-commit": "remove-lockfiles && node index.js",
     "pretest": "npm run lint",
-    "test": "jest --coverage",
-    "deps": "npm-check -s",
-    "deps:update": "npm-check -u"
-  },
-  "pre-commit": "pre-commit",
-  "jest": {
-    "testEnvironment": "node",
-    "setupFiles": [
-      "./testSetup.js"
-    ]
-  },
-  "greenkeeper": {
-    "ignore": [
-      "cosmiconfig"
-    ]
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/okonet/lint-staged.git"
+    "test": "jest --coverage"
```

Comment:
Please add the `test:watch` script. Otherwise I have to do this - `./node_modules/.bin/jest --watch`.

Reply:
cc: @okonet https://github.com/okonet/lint-staged/pull/250#pullrequestreview-61787947

Reply:
See https://github.com/okonet/lint-staged/pull/250#discussion_r138112909. I think he agrees on adding the watch script.

Reply:
I'm for adding this. I'm against of changing CI configs a, though.

Reply:
Updated.. see https://github.com/okonet/lint-staged/pull/250/commits/2330b7cd37c6219caa1625fb1422387e446d5e78